src/router.js

```diff
@@ -31,10 +31,15 @@ export const mountRoutes = (app, provider) => {
         ? [messages[req.query.notification]]
         : [];
 
+      const {
+        session: { info: { acr_values: acr } = {} },
```

Comment:
![image](https://user-images.githubusercontent.com/4190728/68307302-4b3e4080-00ab-11ea-9d6f-63cd357f95f6.png)

do you need help to test this ?

Reply:
As seen together i  have been able to test on my hand, feel free to tell me if you could on yours.